src/components/teasers/grid/teaser-grid.tsx

```diff
@@ -25,13 +39,26 @@ const StyledTeaserGrid = styled.div<{ amountOfChildren: number }>`
   gap: 24px;
 
   justify-items: stretch;
+
+  /**
+   * we give the last element 24px more space for the padding 
+   * -> distance to outer window
+   */
```

Comment:
can you explain with an image? I can't understand the padding. Because of the horizontal scrolling ?

Reply:
On mobile, the `teaser-grid` needs to overflow the pages padding on the right side. 

<img width="174" alt="Screenshot 2021-12-09 at 07 56 15" src="https://user-images.githubusercontent.com/7814926/145348676-1cc3b8d2-c945-4217-bae3-3052513d82fa.png">

To compensate for this overflow, the last element needs to be `24px` larger with a padding of the same size. I tried other solutions but couldn't make it work. Also spent too much time on this already.

Reply:
I'm find with that solution for now